source/kubernetes/kubectl-plugin-reference.rst

```diff
@@ -0,0 +1,650 @@
+  ``/usr/local/bin``.
+
+- For Windows, ensure the path to the ``kubectl-minio`` binary exists in the
+  system ``PATH`` environment variable. Defer to documentation on your version
+  of Windows for instructions on setting the ``PATH`` variable.
+
+2) Validate the Installation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following ``kubectl`` command to validate the installation:
+
+.. code-block:: shell
+
+   kubectl minio operator --version
+
+The operation should return the latest version of the ``kubectl-minio`` plugin.
+
+3) Create the Kubernetes Namespace
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the following ``kubectl`` command to create the namespace for the
+MinIO tenant:
+
+.. code-block:: shell
+   :class: copyable
+
+   kubectl create ns minio-tenant1
+
+4) Create a Kubernetes Secret for MinIO
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+MinIO servers require an access key and secret key for configuring the
+``root`` user. Create a Kubernetes secret for storing these values:
+
+.. code-block:: shell
+   :class: copyable
+
+   kubectl create secret generic minio-tenant1-secret \
+    --from-literal=accesskey=YOUR-ACCESS-KEY \
+    --from-literal=secretkey=YOUR-SECRET-KEY-CHANGE-THIS
+
+5) Create a MinIO Tenant
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Run the :mc-cmd:`kubectl minio tenant create` command to create a MinIO
+tenant on a Kubernetes cluster. The following example uses the
+default ``kubectl`` context:
+
+.. code-block:: shell
+   :class: copyable
+
+   kubectl minio tenant create \
+     --name minio-tenant1
+     --namespace minio \
+     --storageClass direct.csi.min.io \
+     --servers 4 \
+     --volumes 4 \
+     --capacity 16Ti \
+     --secret minio-tenant1-secret
+
+- :mc-cmd-option:`~kubectl minio tenant create servers` sets the total number
+  of ``minio`` servers in the cluster to ``4``. 
+
+- :mc-cmd-option:`~kubectl minio tenant create volumes` sets the total number
+  of volumes per server to ``4``. This results in ``16`` volumes across the
+  cluster. 
+
+- :mc-cmd-option:`~kubectl minio tenant create storageClass` sets the
+  storage class of each volume to ``direct.csi.min.io``. This ensures each
+  generated Persistant Volume Claim (``PVC``) binds to a Direct CSI 
+  Persistant Volume (``PV``).
+
+- :mc-cmd-option:`~kubectl minio tenant create capacity` sets the total
+  MinIO deployment storage capacity to ``16Ti`` or 16 Tebibytes. 
+  ``kubectl-minio`` distributes the capacity evenly over the total nunber of
+  volumes. In this example, the deployment has 16 x ``1Ti`` (1 Tebibyte) drives.
+
+Quick Reference
+---------------
+
+This section contains a quick lookup table of ``kubectl-minio`` commands:
+
+Common Operations
+-----------------
+
+This section lists common operations using the ``kubectl-minio`` plugin:
+
+.. todo
+
+   Need to flesh out a handful of quickstart-like examples here.
+
+Behavior
+--------
+
+.. _kubectl-minio-pvc:
+
+Persistant Volumes and Persistant Volume Claims
```

Comment:
```suggestion
Persistent Volumes and Persistent Volume Claims
```

Reply:
oh blast it all it I knew I was getting that wrong. Will fix throughout.